dcc-submission-loader/pom.xml

```diff
@@ -0,0 +1,151 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright (c) 2016 The Ontario Institute for Cancer Research. All rights reserved. This program and the accompanying 
```

Comment:
Hmm, this really annoys me about the XML formatter. I wonder if there is a fix for that.